piccolo_admin/endpoints.py

```diff
@@ -305,6 +308,9 @@ def __init__(
         rate_limit_provider: t.Optional[RateLimitProvider] = None,
         production: bool = False,
         site_name: str = "Piccolo Admin",
+        logo_image: str = "logo.jpg",
+        logo_width: int = 40,
+        logo_height: int = 30,
```

Comment:
I'm not sure if we need to specify height and width - we can just style the image so it looks OK using CSS. For example:

```css
img {
    height: 1em;
    width: auto;
}
```

Reply:
I agree with you, but I was wondering if someone want a larger logo that we would allow them to do so. If you want, I can remove that option.

Reply:
It's a nice idea, but there's the risk of it breaking the layout if the image is too large. I think we just style it with CSS to make it look nice, without any configuration.

Reply:
Ok. I will remove it